docs_src/microservices/configuration/CommonEnvironmentVariables.md

```diff
@@ -185,4 +211,16 @@ This variable defaults to `1h` (one hour) if unspecified.
 It is often useful when developing a new microservice to set this value to a higher value, such as `12h`.
 This higher value will allow the secret store token to remain valid long enough
 for a developer to get a new microservice working and into a state where it can renew its own token.
-(All secret store tokens in EdgeX expire if not renewed periodically.)
\ No newline at end of file
+(All secret store tokens in EdgeX expire if not renewed periodically.)
+
+#### SECRETSTORE_REVOKEROOTTOKENS (security-secretstore-setup service)
```

Comment:
Please rename this to `EDGEX_SECRETSTORE_REVOKEROOTTOKENS` to be consistent with other env vars in this section.

Reply:
Sorry, didn't realize this was in SecretStore config. It can be removed since it is covered in the SecretStore here: https://docs.edgexfoundry.org/3.0/microservices/configuration/CommonEnvironmentVariables/#secretstore-overrides

Reply:
Just added the one new variable then.